source/_integrations/gandi_livedns.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title: Gandi.net
+description: Keep your computer registered with the gandi live dns dynamic updater.
+ha_category:
+  - Network
+ha_iot_class: "Cloud Polling"
+ha_release: 2021.7
+ha_codeowners:
+  - '@flexy2dd'
+ha_domain: gandi_livedns
+---
+
+With the Gandi.net integration you can keep your current IP address in sync with your [Gandi.net](https://www.gandi.net) registered hostname or domain.
+
+The integration runs at regular intervals, but can also be started manually by using the service `gandi_livedns.update_records` under services.
```

Comment:
This service should be described below, and doesn't really fit as an "introduction" text I think.

Reply:
I have reduced the documentation and reorganized it a bit, i put the essential 
(sorry for the ping-pong party it's my first integration)